test.js

```diff
@@ -26,3 +26,13 @@ test("the reverse", (t) => {
 	const str = "Hallå, Mr. Kebab Überstein! How you doin'?-";
 	t.equal(kebabCase.reverse(kebabCase(str)), str);
 });
+
+test("string with leading dash", (t) => {
+	const str = "KebabCase";
+	t.equal(kebabCase(str, false), "kebab-case");
+})
+
+test("string reverse without leading dash", (t) => {
+	const str = "-kebab-case";
+	t.equal(reverse(str), "KebabCase");
+})
```

Comment:
This test doesn't do what it says... According to the test's title the variable should be:

```ts
const str = "kebab-case";
```

But that will give the result:

```ts
t.equal(reverse(str), "kebabCase");
```

Which is not the reverse of what `kebabCase` outputs with `leadingDashFlag=false`. 
That's my concern about this feature. So that needs to be documented in the readme along with the feature/flag itself.

Reply:
There's also a missing newline at the end of the file...